README.md

```diff
@@ -54,122 +70,28 @@ In [TasksPresenter.java](https://github.com/googlesamples/android-architecture/b
 
-#### Conceptual complexity
+The use of the loaders and content providers introduces additional complexity in writing unit tests for the new functionality. This sample relies on the use of tests which need to be run on either a device or emulator.
 
-Developers need to be familiar with the Loaders and Content Providers framework, which is not
-trivial. Following this approach is harder to decouple from dependencies of the Android Framework.
+## Maintaining the app
 
-### Testability
+To add or amend features to this version of the application, you should be familiar with the implementation details for loaders and content providers, both of which are in depth topics.
 
-#### Unit testing
-
-The use of the Loaders framework adds a big dependency with the Android
-framework so unit testing is harder. Same goes for the Content Provider, forcing us to use AndroidJUnit
-tests which run in a device / emulator and not in the developer's local JVM.
-
-#### UI testing
-
-No difference with MVP.
-
-### Code metrics
-
-Compared to MVP, the only new classes are LoaderProvider and TasksProvider. Parts of
-the code are simpler as Loaders take care of the asynchronous work.
-
-
-```
--------------------------------------------------------------------------------
-Language                     files          blank        comment           code
--------------------------------------------------------------------------------
-Java                            48           1100           1460           3535 (3450 in MVP)
-XML                             34             97            337            601
--------------------------------------------------------------------------------
-SUM:                            82           1197           1797           4136
--------------------------------------------------------------------------------
-
-```
+Note that some aspects of the code in this version are simpler as loaders take care of asynchronous requests. The following table summarizes the amount of code required for this version of the application and provides a comparison to the todo-mvp base sample.
 
-### Maintainability
+| Language      | Number of files | Blank lines | Comment lines | Lines of code |
+| ------------- | --------------- | ----------- | ------------- | ------------- |
+| **Java**      |               48|         1100|           1460|           3535 (3450 in todo-mvP)|
+| **XML**       |               34|           97|            337|            601|
+| **Total**     |               82|         1197|           1797|           4136|
 
-#### Ease of amending or adding a feature
 
-Similar to MVP Loaders
+## Comparing this sample
 
-#### Learning cost
+The following summary reviews how this solution compares to the todo-mvp base sample:
 
-Medium as the Loaders and Content Providers framework are not trivial.
+ * <b>Use of architectural frameworks, libraries, or tools: </b>None required.
```

Comment:
This is based on the original:

Use of architectural frameworks/libraries/tools:

No external frameworks.

I've updated it to read 'Loaders and content providers.'